src/libraries/System.Private.CoreLib/src/System/CodeDom/Compiler/IndentedTextWriter.cs

```diff
@@ -141,6 +159,36 @@ public override void Write(string format, params object?[] arg)
             _writer.Write(format, arg);
         }
 
+        public override async Task WriteAsync(char value)
```

Comment:
I believe all of this new public APIs also need to be added in ref https://github.com/dotnet/runtime/blob/8d7e074fe88d05a66f70e1717c1bea490cd41d2e/src/libraries/System.Runtime/ref/System.Runtime.cs#L5148-L5190

Reply:
Oh okay. I wasn't sure if that was necessary because they're virtual methods defined on a base class, which already has the refs. I'll go ahead and add them.

Reply:
Okay, I've added them.